src/base/compute/owl_computation_graph.ml

```diff
@@ -5,55 +5,50 @@
 
 open Owl_graph
 
-module Make
-  (Optimiser : Owl_computation_optimiser_sig.Sig)
-  = struct
-
+module Make (Optimiser : Owl_computation_optimiser_sig.Sig) = struct
   module Optimiser = Optimiser
-
   open Optimiser
-
   open Optimiser.Operator.Symbol
-
   open Optimiser.Operator.Symbol.Shape.Type
-
   open Optimiser.Operator.Symbol.Shape.Type.Device
 
-
-  type graph = {
-    mutable name   : string;                         (* name of the graph *)
-    mutable input  : attr node array;                (* input nodes of the graph *)
-    mutable output : attr node array;                (* output nodes of the graph *)
-    mutable iopair : (attr node * attr node) array;  (* input and output loopback pairs *)
-    mutable iosafe : bool array;                     (* whether it is safe to use unsafe_assign_arr *)
-    mutable random : attr node array;                (* rvs automatically invalidate themselves *)
-    mutable htbl   : (string, attr node) Hashtbl.t;  (* node name to node mapping *)
-    mutable device : device                          (* device-dependent field *)
-  }
-
+  type graph =
+    { mutable name : string
+    ; (* name of the graph *)
```

Comment:
The positioning of `;` looks really weird. I was thinking if it's possible to just keep it at the end of a line.

Reply:
My latest commit fixes that. I agree that with the comments,  it does look a bit weird; otherwise, I do normally prefer `;` at the front.